README.md

```diff
@@ -1,41 +1,53 @@
 # Openmediavault LDAP
 
-This plugin connects OMV5 to an OPEN-LDAP Server for centralized user management
+This plugin connects OMV6 to an OPEN-LDAP Server for centralized user management
 and authentification. It adapts the official LDAP plugin [1] for OMV4 to use it
-with OMV5.  
-Basically there aren't any fundamental changes up to 5.3.9-r1 tag. You can use LDAP
+with OMV6 or OMV5.  
+Basically there aren't any fundamental changes up to 6.3.5-2 tag. You can use LDAP
 for system wide User- and Groupmanagement. Additionally you can activate PAM auth 
 with *libpam-ldapd* module. The name switch service is done with *libnss-ldapd*.
 For caching actually the NSCD service is used, maybe we should change from NSLCD to 
 SSSD in future. A good explanation of the pam auth and nss process will give
 you the reference [4].  
+The security is enforced at StartTLS or SSL. 
 This repo is under development. Checkout the tags in the master branch for stable 
 and tested releases. If you encounter some problems make a new issue or write an email
 to *devel[at]nareo.de*.  
-Be aware: This code was tested in a clean environment with fresh installed OMV5 setup.
+Be aware: This code was tested in a clean environment with fresh installed OMV6 setup.
 I am not responsible for loss of your data! Please make always a full backup
 of your OMV machine before installing this plugin!
 
+## Install dependencies
+
+sudo apt install build-essential git curl debhelper libnss-ldap libpam-ldap smbldap-tools
+
+## Get code
+
+cd /usr/local/src
+
+git clone https://github.com/lucapiccio/openmediavault-ldap.git
```

Comment:
We should not include this instruction. It depends on the repository and the user preferences.